### ISSUE TITLE
Should "%" times "m" result in "m" by default?

### ISSUE THREAD

**Issue by kendonB** [r-quantities/units#188]
``` r
library(units)
#> udunits system database from C:/R/Library/3.5/units/share/udunits
as_units(10, "%") * as_units(10, "m")
#> 100 [%*m]
```

<sup>Created on 2019-01-22 by the [reprex package](https://reprex.tidyverse.org) (v0.2.1.9000)</sup>

I would have expected to see:
``` r
as_units(10, "%") * as_units(10, "m")
#> 1 [m]
```
Of course, this is a general issue for all continuous units where 0 is meaningful.

**Comment by edzer** [comment]
The issue is really whether `10 [%]` should automatically simplify to `0.1 [1]`, since
```r
> set_units(0.1, 1) * set_units(10, m)
1 [m]
```
There are earlier threads where we discussed this, and concluded that maximal simplification is not what most users will want.

**Comment by Enchufa2** [comment]
And still, `[%*m]` seems odd...

**Comment by kendonB** [comment]
I agree that most users would not want to automatically simplify % to 1. But when multiplying with a measurement unit like above? Surely no one is interested in how many % kgs something is?
